container/standalone/src/main/distribution/etc/system.properties

```diff
@@ -158,3 +153,14 @@ karaf.secured.services = (&(osgi.command.scope=*)(osgi.command.function=*))
 
 # OPENNMS: Enable SNMP4J Log4j logging
 snmp4j.LogFactory=org.snmp4j.log.Log4jLogFactory
+
+# OPENNMS: Disable the Bouncy Castle security provider for Apache MINA SSHD
+# because it silently fails if Bouncy Castle is in the system classpath
+#
+# @see https://github.com/apache/mina-sshd/blob/master/sshd-core/src/main/java/org/apache/sshd/common/util/security/SecurityUtils.java
+#
+#org.apache.sshd.security.registrars = \
+#  org.apache.sshd.common.util.security.bouncycastle.BouncyCastleSecurityProviderRegistrar,\
+#  org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar
+#
+org.apache.sshd.security.registrars = org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar
```

Comment:
I think we'll just need to remove BouncyCastle instead. We've had some issues with it in the past: https://issues.opennms.org/browse/NMS-7959

Reply:
BC is required for the following enhancement because it is a JRadius dependency and JRadius uses it to do EAP-TTLS (afaik from looking at the imports in JRadius):

https://issues.opennms.org/browse/NMS-9416

Maybe we should shade JRadius ourselves to include BC?

Reply:
As discussed, avoiding issues like those in NMS-7959 is more important then supporting the feature in NMS-9416. We should probably just exclude BC from JRadius, and users can drop it in lib/ if they need it.